grafana-plugin/src/containers/AddResponders/parts/AddRespondersPopup/AddRespondersPopup.tsx

```diff
@@ -56,20 +56,19 @@ const AddRespondersPopup = observer(
     const isCreateMode = mode === 'create';
 
     const [activeOption, setActiveOption] = useState<TabOptions>(isCreateMode ? TabOptions.Teams : TabOptions.Users);
+    const [userSearchResults, setUserSearchResults] = useState<UserCurrentlyOnCall[]>([]);
     const [searchTerm, setSearchTerm] = useState('');
 
     const ref = useRef();
     const teamSearchResults = grafanaTeamStore.getSearchResult();
 
-    let userSearchResults = userStore.getSearchResult().results || [];
```

Comment:
lets store the user search results here in `useState` as opposed to `userStore.searchResults` because I want to avoid mixing different types there (`User` vs `UserCurrentlyOnCall`)